### PR TITLE
Add weather

### PR DESCRIPTION
**Pull request by archetipico**
Added weather to script.
Changed the way a distribution is showed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,72 @@
...
+            <img src="https://img.shields.io/github/issues/MasterCruelty/eMerger" alt="issues open">
+            <img src="https://img.shields.io/github/languages/top/MasterCruelty/eMerger" alt="top language">
+        </td>
+        <td>
+            <img src="https://badgen.net/github/release/MasterCruelty/Updater?label=Latest%20release" alt="latest release">
+            <img src="https://img.shields.io/github/commit-activity/w/MasterCruelty/eMerger" alt="commit activity"><br>
+            <img src="https://img.shields.io/github/contributors/MasterCruelty/eMerger" alt="contributors">
+            <img src="https://img.shields.io/github/license/MasterCruelty/eMerger" alt="license">
+            <img src="https://img.shields.io/github/forks/MasterCruelty/Updater" alt="forks">
+            <img src="https://img.shields.io/github/stars/MasterCruelty/Updater" alt="stars">
```
Comment (MasterCruelty) [inline README.md:17]: @archetipico added a couple of things you forgot during the change

File: src/emerger.sh [hunk 2]
```diff
@@ -19,8 +19,16 @@ printf "$LOGO"
...
+printf "Contribute @ https://github.com/MasterCruelty/eMerger $WHALE\nRunning on: "
+
+if [[ -f "/etc/os-release" ]]; then
+    NAME=$(cat /etc/os-release | head -n 7 | tail -n 1 | cut -c 14-)
+    printf "${NAME::-1}\n"
+else
+    printf "$(uname -rs)\n"
+fi
+
+printf "$(curl -s wttr.in/?format="%l:+%c+%t+%w+%m")$NORMAL\n\n"    # wttr.in function
```
Comment (MasterCruelty) [inline src/emerger.sh:31]: @archetipico how this function via curl works exactly?

File: README.md [hunk 3]
```diff
@@ -1,36 +1,70 @@
...
+        <th>Code</th>
+        <th>Details</th>
+    </tr>
+        <td>
+            <img src="https://img.shields.io/github/languages/code-size/MasterCruelty/eMerger" alt="code size">
+            <img src="https://img.shields.io/github/issues/MasterCruelty/eMerger" alt="issues open">
+            <img src="https://img.shields.io/github/languages/top/MasterCruelty/eMerger" alt="top language">
+        </td>
+        <td>
+            <img src="https://img.shields.io/github/commit-activity/w/MasterCruelty/eMerger" alt="commit activity"><br>
```
Comment (MasterCruelty) [inline README.md:12]: Ok autism satisfied

**Review by archetipico — COMMENTED** [review]
No problem

**Review by MasterCruelty — COMMENTED** [review]
@archetipico can you explain how the weather function works?

File: src/emerger.sh (hunk 2 above)
Comment (archetipico) [reply src/emerger.sh:31]: The Inquisition has arrived

Comment (archetipico) [reply src/emerger.sh:31]: Try to print `curl -s wttr.in/?format="%l:+%c+%t+%w+%m"` on your terminal. It retrieves data from [wttr.in](wttr.in) and it displays it in your terminal

Comment (MasterCruelty) [reply src/emerger.sh:31]: Ok cool 🚀 tested now and it's working.
Another question then, can you say me more about the arguments you pass? For example ```%1``` or ```%c``` and the others

Comment (archetipico) [reply src/emerger.sh:31]: Yes you can find your answer [here](https://en.wikipedia.org/wiki/RTFM).

Jk.
%l = location
%c = I suppose it stands for climate and it shows the weather emoji
%t = temperature (actual, not felt)
%w = wind
%m = Moon phase

Comment (MasterCruelty) [reply src/emerger.sh]: I added these information as comment in the source file

**Review by MasterCruelty — APPROVED** [review]
Review completed I guess 👓 
WE HAVE WEATHER NOW

**Pull request merged**